### PR TITLE
vmui: option to see/copy the full error message

### PR DESCRIPTION
**Pull request by Loori-R**
Changed the display logic for long errors in the text field.
On the click on err msg, display the full text of the message.

#4719

demo_close
demo_open

### CODE REVIEW

**Review by dmitryk-dk — APPROVED** [review]

**Review by hagen1778 — COMMENTED** [review]
Thanks for implementation!
I have the following concerns with it:
1. When user sees the full msg in tooltip and wants to copy it, they would intuitively move cursor to the text in tooltip. Once focus is lost, the popup will disappear and user's attempt to copy text will fail.
2. Looking for alternative, user may (or may not) click on message and got a notification about copied text. This is nice, but not intuitive from the first try
3. There is a glitch for text displayed in popup before and after click on error message: for a short time tooltip will contain `copied` message even if focus was lost and re-established again.


What I think would be simpler to implement and support is the following:
1. On  the click on err msg, display the full text of the message right there without word breaking (as it was before we introduced msg cropping).
2. Once msg displayed, user can decide what to do: copy full msg, part of it or make a screenshot
3. On expr re-evauluation error msg behavior should return to default, collapsed state.
I believe the described approach should be simpler to implement and more intuitive for a user to follow.

**Comment by Loori-R** [comment]
@hagen1778, updated, check please

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -34,6 +34,7 @@ The following `tip` changes can be tested by building VictoriaMetrics components
 * FEATURE: [vmctl](https://docs.victoriametrics.com/vmctl.html): add support of `week` step for [time-based chunking migration](https://docs.victoriametrics.com/vmctl.html#using-time-based-chunking-of-migration). See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4738).
 * FEATURE: [vmctl](https://docs.victoriametrics.com/vmctl.html): do not add `/api/v1/read` suffix to remote read storage address defined by `--remote-read-src-addr` if a `--remote-read-disable-path-append` command-line flag is set. It allows an overriding path for remote-read API via `--remote-read-src-addr`. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4655).
 * FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add warning in query field of vmui for partial data responses. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4721).
+* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): add the ability to view and copy the text of a lengthy error in a text field. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4719).
```
Comment (hagen1778) [inline docs/CHANGELOG.md:37]:
```suggestion
* FEATURE: [vmui](https://docs.victoriametrics.com/#vmui): allow displaying the full error message on click for trimmed error messages in vmui. See [this issue](https://github.com/VictoriaMetrics/VictoriaMetrics/issues/4719).
```

**Review by hagen1778 — APPROVED** [review]
LGTM!

**Pull request merged**